Recorder/OnboardingView.swift

```diff
@@ -0,0 +1,86 @@
+//
+//  OnboardingView.swift
+//  Recorder
+//
+//  Created by baek seohyeon on 2022/06/16.
+//
+
+import SwiftUI
+
+//초기 시작화면을 결정하는 뷰
+struct OnboardingStartView: View{
+    
+    //최초 실행을 관리하는 앱스토리지
+    @AppStorage("_isFirstLaunching") var isFriestLaunching: Bool = true
+    
+    //스타트 버튼을 누르기 전까지는 else. 즉 온보딩 뷰가 보임
+    @State var StartButton: Bool = false
+    var body: some View {
+        if !isFriestLaunching {
+            HomeView()
+        } else {
+            OnboardingView(isFriestLaunching: $isFriestLaunching)
+        }
+    }
+}
+
+//온보딩 뷰
+struct OnboardingView: View {
+    
+    @Binding var isFriestLaunching: Bool
+    
+    //페이징 수를 결정하는 배열과 변수
+    @State var selected = 0
+    var pageIndex : [Int] = [0, 1, 2, 3]
+    
+    let title : [String] = ["나의 이야기로\n간직하고 싶은 음악이 있나요?","음악을 사진과 함께\n짧은 이야기로 기록하고,\n기억하고 싶은 가사를 입력하세요.","나만의 방에서 \n음악을 플레이어에 재생시키고 \n기록을 다시 감상하세요."]
+    let backgroundImage : [String] = ["Page1","Page2","Page3"]
```

Comment:
이렇게 하면 모두가 편안한 코드 규칙입니다!
```suggestion
    var pageIndex: [Int] = [0, 1, 2, 3]
    
    let title: [String] = ["나의 이야기로\n간직하고 싶은 음악이 있나요?","음악을 사진과 함께\n짧은 이야기로 기록하고,\n기억하고 싶은 가사를 입력하세요.","나만의 방에서 \n음악을 플레이어에 재생시키고 \n기록을 다시 감상하세요."]
    let backgroundImage: [String] = ["Page1","Page2","Page3"]
```